aries/wrappers/uniffi-aries-vcx/core/Cargo.toml

```diff
@@ -16,9 +16,11 @@ path = "uniffi-bindgen.rs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["vdrtools_wallet"]
-vdrtools_wallet = ["aries_vcx/vdrtools_wallet"]
+default = ["main_vdrtools_wallet"]
 askar_wallet = ["aries_vcx/askar_wallet"]
+vdrtools_wallet = ["aries_vcx/vdrtools_wallet"]
+main_vdrtools_wallet = ["vdrtools_wallet"]
```

Comment:
Seems like feature alias? I'd rather keep it simple, just keep the original feature flag names.